inappmessaging/src/test/java/com/rakuten/tech/mobile/inappmessaging/runtime/utils/InitializerSpec.kt

```diff
@@ -44,14 +47,10 @@ class InitializerSpec : BaseTest() {
     @Test
     fun `should add host app info with basic attributes`() {
         WorkManagerTestInitHelper.initializeTestWorkManager(context!!)
-        InAppMessaging.init(
-                context,
-                "test_sub_key",
-                "",
-                isDebugLogging = false,
-                isForTesting = true)
+        InAppMessaging.initialize(context, true)
         InAppMessaging.instance().registerPreference(TestUserInfoProvider())
-        HostAppInfoRepository.instance().getInAppMessagingSubscriptionKey() shouldBeEqualTo "test_sub_key"
+        HostAppInfoRepository.instance()
```

Comment:
Can we add a check if `getInAppMessagingSubscriptionKey` is not empty at least? (Just in case subscriptionKey() in manifest is also empty for some reason)

Reply:
Added null/empty checking